applications/connecting_applications_to_services/binding-workloads-using-sbo.adoc

```diff
@@ -0,0 +1,47 @@
+[id="binding-workloads-using-sbo"]
+= Binding workloads using Service Binding Operator
+include::modules/common-attributes.adoc[]
+include::modules/servicebinding-document-attributes.adoc[]
+:context: binding-workloads-using-sbo
+
+toc::[]
+
+Application developers must bind a workload to one or more backing services using a binding secret generated for the purpose of storing information to be consumed by the workload.
```

Comment:
```suggestion
Application developers must bind a workload to one or more backing services by using a binding secret. This secret is generated for the purpose of storing information to be consumed by the workload.
```
This sentence feels like there's a bit too much information in it. Consider creating two sentences for the content. Above is one suggestion for this edit.